### PR TITLE
Remove doorkeeper

### PR DESCRIPTION
**Pull request by sonalkr132**
Requested [here](https://github.com/rubygems/rubygems.org/pull/1405#issuecomment-247065634).

> The app that was using oauth is not in use at all right now. So nothing is using oauth/doorkeeper at the moment. I think it would be best to remove it for now. We can always add it back later if we need it again. But I don't think there are any plans to use it, and it's better to remove unused things for the health of the project.

### CODE REVIEW

**Comment by dwradcliffe** [comment]
cc: @bf4 @qrush

File: db/schema.rb [hunk 1]
```diff
@@ -213,9 +173,9 @@
     t.string   "required_ruby_version"
     t.string   "sha256"
     t.hstore   "metadata",                  default: {},   null: false
+    t.datetime "yanked_at"
```
Comment (simi) [inline db/schema.rb]: :question:

Comment (dwradcliffe) [inline db/schema.rb]: It's been doing this on my local copy too, guessing it's just committed in the wrong order?

Comment (simi) [inline db/schema.rb]: let me try

Comment (simi) [inline db/schema.rb]: ok, same here

**Comment by andrew** [comment]
This is a shame that all the oauth functionality has been removed, I was hoping to use it so that Libraries.io users could prove who they are on rubygems.org

At the moment the only way to know if a person is that user on rubygems is with email verification which is still a bit crappy, rubygems was one of the only package managers that had oauth and I was hoping to use it as an example to encourage other package managers to do the same so we can do more things like adoption projects.

**Comment by bf4** [comment]
@andrew are you more interested in verifying ownership, i.e. read once, or some tighter integration with write?  How would you feel about a user-generated one-time token?

**Comment by andrew** [comment]
@bf4 read only access and I think one-time would do fine as all the owner details are public and available via the api already

**Pull request merged**